types/sonic-boom/sonic-boom-tests.ts

```diff
@@ -1,4 +1,4 @@
-import SonicBoom = require('sonic-boom');
```

Comment:
To verify the correct export (see my other comment) you have to leave that line `import SonicBoom = require('sonic-boom');` as it was before.